modules/machineconfig-modify-registry.adoc

```diff
@@ -0,0 +1,98 @@
+// Module included in the following assemblies:
+//
+// * installing/post_installation_configuration/machine-configuration-tasks.adoc
+
+[id="machineconfig-modify-registry_{context}"]
+= Configuring container image registry settings
+
+Settings that define the registries {product-title} uses to get container images
+are held in the `/etc/containers/registries.conf` by default. In that file,
+you can set registries to not require authentication (insecure), point to
+mirrored registries, or set which registries are searched for unqualified
+container image requests.
+
+Rather than change `registries.conf` directly, you can drop configuration files
+into the `/etc/containers/registries.d` directory that are then automatically
+appended to the system's existing `registries.conf` settings.
+
+This procedure describes how to create a `registries.d` file
+(`/etc/containers/registries.s/99-worker-unqualified-search-registries.conf`)
+that adds `quay.io` as an unqualified search registry (one that {product-title}
+can search when it tries to pull an image name does not include the registry name).
+It includes base64-encoded content that you can examine as follows:
+
+[source,terminal]
+----
+$ echo dW5xdWFsaWZpZWQtc2VhcmNoLXJlZ2lzdHJpZXMgPSBbJ3JlZ2lzdHJ5LmFjY2Vzcy5yZWRoYXQuY29tJywgJ2RvY2tlci5pbycsICdxdWF5LmlvJ10K | base64 -d
+unqualified-search-registries = ['registry.access.redhat.com', 'docker.io', 'quay.io']
+----
+
+See the `containers-registries.conf` man page for the format for the
+`registries.conf` and `registries.d` directory files.
+
+.Prerequisites
+* Have a running {product-title} cluster (version 4.4 or later).
+* Log in to the cluster as a user with administrative privileges.
+
+.Procedure
+
+. Create a yaml file (`myregistry.yaml`) to hold the contents of the
+`/etc/containers/registries.d/99-worker-unqualified-search-registries.conf`
+file, including the encoded base64 contents of the for that file. For example:
```

Comment:
there's an extra preposition here: "of the for that file"

Reply:
Fixed.